docs/jackson/deploy.md

```diff
@@ -21,14 +21,79 @@ npm i @boxyhq/saml-jackson
 
 ## As a separate service
 
-The docker container can be found at [boxyhq/jackson](https://hub.docker.com/r/boxyhq/jackson/tags). It is preferable to use a specific version instead of the `latest` tag. Jackson uses two ports (configurable if needed, see below) 5000 and 6000. 6000 is the internal port and ideally should not be exposed to a public network.
+### With Docker
+
+The docker container can be found at [boxyhq/jackson](https://hub.docker.com/r/boxyhq/jackson/tags). It is preferable to use a specific version instead of the `latest` tag.
+
+```bash
+docker run \
+  -p 5000:5000 \
+  -e JACKSON_API_KEYS="secret" \
+  -e DB_ENGINE="sql" \
+  -e DB_TYPE="postgres" \
+  -e DB_URL="postgres://postgres:postgres@postgres:5432/postgres" \
+  -e DB_ENCRYPTION_KEY="fwTul69Mf0nZDr+L1SG9gqwcOkyieybu" \
```

Comment:
Let's not include any encryption keys since people tend to copy paste and forget about it